### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by renefritze**

### CODE REVIEW

**Comment by renefritze** [comment]
You should be able to see the sphinx output [here](https://readthedocs.org/projects/pytimings/builds/13664140/) soon.

File: docs/examples/scopes_and_decorators.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```{code-cell}
+
+from pytimings.timer import global_timings
+
+global_timings.output_console()
+```
+
+You can see that the `my_function` timer recorded all 4 calls,
+while the `my_scope` only containts one `0.5` call.
```
Comment (TiKeil) [inline docs/examples/scopes_and_decorators.md:64]:
```suggestion
while the `my_scope` only contains one `0.5` call.
```

File: docs/examples/basic.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+To inspect deltas of known sections at any other point
+use the `delta` function.
+
+```{code-cell}
+
+global_timings.delta("a_section_name")
+```
+
```
Comment (TiKeil) [inline docs/examples/basic.md:40]: perhaps also mention `global_timings.walltime("a_section_name")` or add some hint that there are more helpful functions in `timer.py`?

**Review by TiKeil — CHANGES_REQUESTED** [review]
Looking pretty tasty. Just found three minor things. 
I think these small examples are very good for getting started.

File: docs/examples/basic.md (hunk 2 above)
Comment (renefritze) [reply docs/examples/basic.md:40]: LInk into the api docs maybe?

**Review by TiKeil — APPROVED** [review]

**Pull request merged**